chain/client/src/client_actor.rs

```diff
@@ -175,6 +179,11 @@ impl Handler<NetworkClientMessages> for ClientActor {
     type Result = NetworkClientResponses;
 
     fn handle(&mut self, msg: NetworkClientMessages, _: &mut Context<Self>) -> Self::Result {
+        // There is a bug in Actix library. While there are messages in mailbox, Actix
+        // will prioritize processing messages until mailbox is empty. Execution of any other task
+        // scheduled with run_later will be delayed.
+        self.try_handle_block_production();
```

Comment:
Shall we also add it to the handlers of `Status` and `GetNetworkInfo` messages below?

Reply:
agree